examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.2_identity_security/identity_security.ipynb

```diff
@@ -0,0 +1,210 @@
+    "\n",
+    "When distributing Startup Kits, the Project Admin should use trusted communication methods, and never send passwords of the Startup Kits in the same communication. It is preferred to send the Kits and passwords with different communication methods.\n",
+    "\n",
+    "Org Admin and users must protect their Startup Kits and only use them for intended purposes.\n",
+    "\n",
+    ">  Note\n",
+    "The provisioning tool tries to use the strongest cryptography suites possible when generating the PKI credentials. All of the certificates are compliant with the X.509 standard. All private keys are generated with a size of 2048-bits. The backend is openssl 1.1.1f, released on March 31, 2020, with no known CVE. All certificates expire within 360 days.\n",
+    "\n",
+    "> NVFlare Dashboard is a website that supports user and site registration. Users will be able to download their Startup Kits (and other artifacts) from the website.\n",
+    "\n",
+    "Let's take a look above in action\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, let's generate a default startup kits using POC mode. POC internally call provision to generate the default startup kits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! echo y | nvflare poc prepare\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! tree /tmp/nvflare/poc/example_project/prod_00"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Look at the each site's start up kit. We see\n",
+    "\n",
+    "```\n",
+    "startup\n",
+    "│   │   ├── client.crt  <-- certificate\n",
+    "│   │   ├── client.key  <-- private key\n",
+    "│   │   ├── fed_client.json\n",
+    "│   │   ├── rootCA.pem  <-- root certificate\n",
+    "│   │   ├── signature.json\n",
+    "│   │   ├── start.sh\n",
+    "│   │   ├── stop_fl.sh\n",
+    "│   │   └── sub_start.sh\n",
+    "\n",
+    "```\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Authorization\n",
+    "\n",
+    "Federated learning is conducted over computing resources owned by different organizations. Naturally these organizations have concerns about their computing resources being misused or abused. Even if an NVFLARE docker is trusted by participating orgs, researchers can still bring their own custom code to be part of a study, which could be a big concern to many organizations. In addition, organizations may also have IP (intellectual property) requirements on the studies performed by their own researchers.\n",
```

Comment:
suggest add "can still bring their own custom code (BYOC) ..."

to be connected with the next paragraph